python/tests/unit/arcticdb/version_store/test_update.py

```diff
@@ -450,7 +450,7 @@ def _create_product_candles_df(arr):
     assert_frame_equal(after_arctic, before_arctic)
 
 
-def test_update_non_sorted_exception(lmdb_version_store):
+def test_update_new_non_sorted_exception(lmdb_version_store):
```

Comment:
What does "new" mean in this context?

Reply:
Try `test_update_with_unsorted_input_throws`?

Reply:
well otherwise the test would fail isn't it

Reply:
I changed the name